app/controllers/finders_controller.rb

```diff
@@ -8,6 +8,16 @@ class FindersController < ApplicationController
   def show
     slimmer_template "gem_layout_full_width" if i_am_a_topic_page_finder
 
+    ab_test = GovukAbTesting::AbTest.new(
```

Comment:
The AB test will run on every page shown by this action, which might not be desired if K&C just want results from /search/all.  
You could add a `page_under_test?` method that tests whether the path is `/search/all`, and wrap this logic in that?

Reply:
Thanks, I checked with them and they only want it on /search/all... which makes sense as this is what they will want to test with the real AB test! Thanks for flagging. I have updated it to use the constraint.